### PR TITLE
KnowledgeBase merge

### PR DESCRIPTION
**Pull request by bailbot**

### CODE REVIEW

File: Database-Features/server-timezone-jdbc-connection.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+I'm using a LUA function to pull data from a mysql database. 
+
+MySQL database operates in UTC and Exasol Server is on America/New York . The function uses the mysql function from_unixtime() to convert from utc unixtime to utc datetime format but it looks like when I pull from the UDF, the datetime is already converted to the server timezone when I really I need to be storing UTC timestamps. 
+
+Is there a way that I can set outside my function or without hardcoding the timezone conversion (as it is a function that is used in multiple environments) to make sure that the timezone is honored?
+
+## Answer
+Do you use a new JDBC-Connection To MySQL in the UDF? You could then overwrite the server-timezone with the JDBC-param to your own timezone, thereby preventing the conversion:
+
+&serverTimezone=\<YOURTIMEZONE>
```
Comment (nconforti93) [inline Database-Features/server-timezone-jdbc-connection.md:13]: Use "&lt;" and "&gt;" for the characters

File: Database-Features/wrong-results-connect-by-nocycle.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+|7    |
+|8    |
+|9    |
+
+In fact, without any usage of PRIOR, I think that NOCYCLE can be safely ignored?
+
+## Answer
+As per Oracle's definition a loop occurs if one row is both the parent (or grandparent or direct ancestor) and a child (or a grandchild or a direct descendant) of another row. This is independent of the PRIOR condition but just depends on the connect by condition: in your case of LEVEL < 10. The same single row of DUAL is joined repeatedly while executing the connect by. Since there is only one row this leads to a cycle because the row with level 2 has the same row as parent and as child.
+
+I am not sure why Oracle implemented this differently. That said we are not compatible to Oracle when it comes to cycle detection (see https://community.exasol.com/t5/database-features/connect-by-cycle-detection/ta-p/1666).
```
Comment (nconforti93) [inline Database-Features/wrong-results-connect-by-nocycle.md:40]: Can you put the new link already? Search on https://exasol.my.site.com/s/?language=en_US

File: Environment-Management/cloud-ami-list.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# List of Available Cloud AMIs
+
+## Question
+if I´m not mistaken then currently when we use https://cloudtools.exasol.com/ to create a new exasol cluster we will
```
Comment (nconforti93) [inline Environment-Management/cloud-ami-list.md:4]: Funny character in i'm

File: Environment-Management/restart-cloud-ui-service.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+server = xmlrpc.client.ServerProxy  
+('https://admin:<password>@<ip_or_dns>/cluster1', context=ssl._create_unverified_context())  
+#Test if the Plugin was uploaded correctly  
+server.showPluginList()  
+--['Cloud.UIBackend-1.1.3']  
+server.callPlugin('Cloud.UIBackend-1.1.3', 'n0010', 'RESTART', '')
+```
+2. If the XML-RPC call does not work please try to restart it from systemd. Login to the license (management) server via SSH and run "systemctl restart cloudui" command.
+
+After restarting the Cloud UI service please restart the Exaoperation service. You can do this via Exaoperation WEB UI -Exaoperation -Restart
```
Comment (nconforti93) [inline Environment-Management/restart-cloud-ui-service.md:30]: The formatting here looks wrong. Is it supposed to be like -Exaoperation or -Restart ?

**Review by nconforti93 — CHANGES_REQUESTED** [review]
See my comments on the files

File: Data-Science/can-lua-set-autocommit-off.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Can LUA Set AutoCommit Off?
```
Comment (allipatev) [inline Data-Science/can-lua-set-autocommit-off.md:1]: Does it really belong to "Data Science" section?

File: Data-Science/cast-ignored-virtual-schema.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+# Cast Ignored When Querying Virtual Schema
```
Comment (allipatev) [inline Data-Science/cast-ignored-virtual-schema.md:1]: Does it really belong to "Data Science" section?

File: Data-Science/cast-ignored-virtual-schema.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+## Question
+I'm running two select statements on virtual schema's. Since they're joined in a union and don't all have the same columns, I need a column with '' or NULL values. This results in an error: SQL Error [0A000]: Feature not supported: datatypes are not compatible for Union (Session: 1715323572262273024)
+
+The '' or NULL column appears to be a boolean.
+
+Changing this to the desired data type using: CAST( '' AS VARCHAR(18)) results in exactly the same error.
+
+Running the statement separately shows that the CAST is ignored. It appears to happen when querying on virtual schema's. Doing the same on regular schema's or without referring to objects does not result in the CAST being ignored.
+
+Is there a way to make my join work without creating additional objects?
```
Comment (allipatev) [inline Data-Science/cast-ignored-virtual-schema.md:12]: join -> query?

File: Data-Science/cast-ignored-virtual-schema.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
+# Cast Ignored When Querying Virtual Schema
+
+## Question
+I'm running two select statements on virtual schema's. Since they're joined in a union and don't all have the same columns, I need a column with '' or NULL values. This results in an error: SQL Error [0A000]: Feature not supported: datatypes are not compatible for Union (Session: 1715323572262273024)
+
+The '' or NULL column appears to be a boolean.
+
+Changing this to the desired data type using: CAST( '' AS VARCHAR(18)) results in exactly the same error.
+
+Running the statement separately shows that the CAST is ignored. It appears to happen when querying on virtual schema's. Doing the same on regular schema's or without referring to objects does not result in the CAST being ignored.
```
Comment (allipatev) [inline Data-Science/cast-ignored-virtual-schema.md:10]: schema's -> schemas?
2 times

**Pull request closed without merging**